src/utilities/introspectionQuery.js

```diff
@@ -13,10 +13,15 @@ export type IntrospectionOptions = {|
   // Whether to include descriptions in the introspection result.
   // Default: true
   descriptions: boolean,
+  // Whether to include `isRepeatable` flag on directives.
+  // Default: false
```

Comment:
Why would this default to false?

Reply:
For example, if it would be `true`, then after an update without any adjustments third-party libraries (like GraphiQL) would start to send queries with `isRepeatable` flag to the servers that potentially not updated yet. So default is primarily defined like this to maintain backwards compatibility.

Would you like to change the default and require all downstream libraries to explicitly set this flag? (at least at the initial stage where servers are not yet updated)

Reply:
@mjmahone Do you remember we discussed it on the last WG in the scope of my 
"schema description" RFC. And the consensus was to keep it backward compatible for now but design two stage introspection as a long-term solution.